app/assets/javascripts/cascade/level/level.js

```diff
@@ -16,6 +16,8 @@ function wrapIframes() {
   var notSelector = iframeSelectorBlacklist.join(', ');
 
   $('.editableContent iframe').not(notSelector).each(function() {
+    //Ensures that embedded map iframe elements do not receive a div element with class video
+    if ($(this)[0].id === "map_frame") return
```

Comment:
Does it not work if you just add `#map_frame` to `iframeSelectorBlacklist` on line 9? Seems like this is what it was added for, so we should try to use it.

Reply:
I just tested it and your right it does prevent it from adding the video div around the element. I've removed this conditional and added it to the blacklist.